.github/workflows/open-source-unit-tests.yml

```diff
@@ -20,12 +20,10 @@ on:
       image_tag:
         description: 'Marqo image tag. Examples: "1.1.0", "test" "latest"'
         required: true
-  push:
-    branches: 
-      - mainline
```

Comment:
This is removed so that unit tests don't run 3 times for each push to mainline. The other two places the unit tests run, on a push to mainline:
- Once for PR sync action (below)
- Once running the push-to-pypi workflow